cms/djangoapps/contentstore/tests/utils.py

```diff
@@ -299,8 +299,14 @@ def assertCoursesEqual(self, course1_id, course2_id):
                 store = self.store._get_modulestore_for_courselike(course2_id)  # pylint: disable=protected-access
                 new_name = 'course' if isinstance(store, SplitMongoModuleStore) else course2_item_loc.run
                 course2_item_loc = course2_item_loc.replace(name=new_name)
-            course2_item = self.store.get_item(course2_item_loc)
-
+                course2_item = self.store.get_item(course2_item_loc)
+                # The EnrollmentTrackuserPartition stores the course_id, which will vary split vs. old mongo.
```

Comment:
Note to reviewers: cloning an old mongo course into the new modulestore caused me some grief because the course_id at the beginning of the operation is the old ID, and then it changes at some point to the new (split) ID. I updated the course_module code to handle that, and then unit tests calling this method started to fail because the "old" course was not equal to the "new" course.